examples/experimental/bert_squad_pytorch/const.yaml

```diff
@@ -0,0 +1,27 @@
+description: Bert_SQuAD_PyTorch
+hyperparameters:
+    global_batch_size: 24
+    learning_rate: 3e-5
+    lr_scheduler_epoch_freq: 1
+    model_type: 'bert'
+    adam_epsilon: 1e-8
+    weight_decay: 0
+    num_warmup_steps: 0
+    num_training_steps: 400
+    max_seq_length: 384
+    doc_stride: 128
+    max_query_length: 64
+    n_best_size: 20
+    max_answer_length: 30
+    null_score_diff_threshold: 0.0
+searcher:
+    name: single
+    metric: f1
+    max_steps: 2
+    smaller_is_better: false
+min_validation_period: 200 
+data:
+    model_name_or_path: "bert-base-uncased"
```

Comment:
I have two questions based on the naming `model_name_or_path`.

1) is there a scenario in which this would be a `path` instead of a model name, in the context of this Determined example? If not, I would just call it `model_name`.
2) is this used for figuring out how to initialize the weights of this training job? if so, a better name might make that explicit: `pretrained_model_name`.

Reply:
Good point here